nginx.conf

```
@@ -57,12 +59,13 @@ http {
             image_filter resize $width $height;
         }
 
-        # proxy all other requests to the AWS
+        # proxy all other requests to the azure zoonversestatic storage account, $web container
         location / {
             rewrite "^/([1-9][0-9]{0,2}x[1-9][0-9]{0,2})(/.*)$" $2 break;
             resolver 8.8.8.8;
-            proxy_pass http://zooniverse-static.s3-website-us-east-1.amazonaws.com$uri?$query_string;
-            proxy_set_header       Host zooniverse-static.s3-website-us-east-1.amazonaws.com;
+            proxy_http_version 1.1;
+            proxy_pass https://zooniversestatic.z13.web.core.windows.net$uri;
```

I think this should have the query string appended as well 
```suggestion
            proxy_pass https://zooniversestatic.z13.web.core.windows.net$uri?$query_string;
```